website/content/docs/secrets/databases/index.mdx

```diff
@@ -104,75 +105,157 @@ TTL.
+
+```shell-session
+$ vault write database/config/my-database \
+    plugin_name="..." \
+    connection_url="..." \
+    allowed_roles="..." \
+    username="..." \
+    password="..." \
+```
+
+~> It is highly recommended a user within the database is created
+specifically for Vault to use. This user will be used to manipulate
+dynamic and static users within the database. This user is called the
+"root" user within the documentation.
+
+Vault will use the user specified here to create/update/revoke database
+credentials. That user must have the appropriate permissions to perform
+actions upon other database users (create, update credentials, delete, etc.).
+
+This secrets engine can configure multiple database connections. For details
+on the specific configuration options, please see the database-specific
+documentation.
+
+</Tab>
+
+<Tab heading="GUI"  group="gui">
+
+![Partial screenshot of the Vault GUI showing the "Create connection" page](/img/gui/databases/configureDBConnection.png)
+
+1. From Secret Engines List, click on **database/**
+1. Click on **Connections** tab and click **Create connection +**
+1. Click on Database plugin dropdown and select desired plugin ie. **postgreSQL**
+
+![Partial screenshot of the Vault GUI showing the populated "Plugin config" section](/img/gui/databases/pluginConfig.png)
+
+1. Fill in the chosen plugin's specific fields
+1. Click create database to create the database connection
+
+</Tab>
+
+</Tabs>
+
+After configuring the root user, it is highly recommended you rotate that user's
+password such that the vault user is not accessible by any users other than
+Vault itself:
+
+```shell-session
+$ vault write -force database/rotate-root/my-database
+```
+
+!> When this is done, the password for the user specified in the previous step
+is no longer accessible. Because of this, it is highly recommended that a
+user is created specifically for Vault to use to manage database
+users.
 
-   The `{{username}}` and `{{password}}` fields will be populated by the plugin
-   with dynamically generated values. In some plugins the `{{expiration}}` field is also supported.
+<Tabs>
+
+<Tab heading="CLI" group="cli">
+
+Configure a role that maps a name in Vault to a set of creation statements to
+create the database credential:
+
+```shell-session
+$ vault write database/roles/my-role \
+    db_name=my-database \
+    creation_statements="..." \
+    default_ttl="1h" \
+    max_ttl="24h"
+Success! Data written to: database/roles/my-role
+```
+
+The `{{username}}` and `{{password}}` fields will be populated by the plugin
+with dynamically generated values. In some plugins the `{{expiration}}` field is also supported.
+
+</Tab>
+
+<Tab heading="GUI"  group="gui">
+
+1. From Secret Engines List, click on **database/**
+1. Click on **Roles** tab and click **Create role +**
+
+![Screenshot of the Vault GUI showing the "Create Role" page](/img/gui/databases/roleDefaultState.png)
```

Comment:
When we include screenshots, they should be the first item in the tab instead of splitting the instructions